recipe/meta.yaml

```diff
@@ -24,38 +27,37 @@ build:
     - MIGRATING={{ migrating }}
 
 requirements:
-  {% if not win %}
-  build:
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - ipython >=7.23.1                       # [build_platform != target_platform]
-    - jupyter_client >=6.1.12                # [build_platform != target_platform]
-    - packaging                              # [build_platform != target_platform]
-    - psutil                                 # [build_platform != target_platform]
-    - python                                 # [build_platform != target_platform]
-  {% endif %}
   host:
+    {% if unix %}
+    - __unix
+    {% else %}
+    - __win
+    {% endif %}
     - debugpy >=1.0.0
     - ipython >=7.23.1
     - jupyter_client >=6.1.12
     - packaging
     - pip
     - psutil
-    - python
+    - python 3.9
   run:
-    - appnope  # [osx]
+    {% if unix %}
```

Comment:
looks like this can be unconditional as well, because `noarch` (as of today)

Reply:
Fixed, thanks!